.gitignore

```diff
@@ -62,3 +62,7 @@ target/
 # External Libraries 
 wger/core/static/bower_components
 node_modules
+.DS_Store
+venv-django/
+wger/.DS_Store
```

Comment:
Good job ignoring these files. However, I think ignoring `wger/.DS_Store` should have been matched by `.DS_Store`.